wiki/en/Running-a-Server.md

```diff
@@ -92,16 +92,14 @@ If you want to run a Server on a Raspberry Pi, you will need to [compile from so
 
 ### Installation
 
-1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Download the latest headless .deb file: `wget {{ site.download_root_link }}{{ site.download_file_names.deb-headless }}`
 1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
 1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
 1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
-1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file in `/lib/systemd/system/jamulus-headless.service` (By default you will be running an Unregistered Server).
+1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file by running `systemctl edit jamulus-headless` (By default you will be running an Unregistered Server).
 1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
```

Comment:
```suggestion
1. Restart the headless Server: `sudo systemctl restart jamulus-headless`
```

At least, I didn't need to `daemon-reload` it.

Reply:
I think it’s cleaner to do it this way

Reply:
Yes, so no separate "config" needed as systemd effectively creates a local config with `edit`. And if the user upgrades, we don't have to worry about overwriting their stuff.